NitroxServer/GameLogic/Entities/Spawning/BatchEntitySpawner.cs

```diff
@@ -218,11 +231,30 @@ private IEnumerable<Entity> CreateEntityWithChildren(EntitySpawnPoint entitySpaw
 
             yield return spawnedEntity;
 
-            // Children are yielded as well so they can be indexed at the top level (for use by simulation 
-            // ownership and various other consumers).  The parent should always be yielded before the children
-            foreach (Entity childEntity in spawnedEntity.ChildEntities)
+            if (parentEntity == null)
```

Comment:
Why does the spawnedEntity need to be parentless to emit its children?  Does this ensure that we only emit at the very top level (cell root)?  If that is the case, can spawnedEntity above be yielded twice if its cell root were to yield it as a child?  Let's clarify the exact behavior of this condition in a comment.